components/batch-system/src/batch.rs

```diff
@@ -466,6 +474,11 @@ impl<N: Fsm, C: Fsm, Handler: PollHandler<N, C>> Poller<N, C, Handler> {
             }
         }
         batch.clear();
+
+        info!(
```

Comment:
What's the purpose of this log. Same as the one above.

Reply:
It's used for me to debug. I will remove them later.